test/linter/test-descriptions.js

```diff
@@ -6,123 +6,108 @@ const { Logger } = require('./utils.js');
  */
 
 /**
- * @param {Identifier} apiData
- * @param {String} apiName
- * @param {Logger} logger
+ * Check for errors in the description of a specified statement's description and return whether there's an error and log as such
+ *
+ * @param {string} ruleName The name of the error
+ * @param {string} name The name of the API method
+ * @param {Identifier} method The method's compat data
```

Comment:
As established, `method` is actually more general a `feature` now (guess the check was originally used for methods only).

```suggestion
 * @param {Identifier} feature - The feature's compat data.
```